24-Perception/Edge-Detection/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+	<title>Edge Detection - Introduction</title>
+
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+		integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+	<link rel="stylesheet" href="/styles.css">
+	<link rel="stylesheet" href="./style.css">
+
+	<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+	<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
+	<script type="text/javascript" src="/main.js"></script>
+</head>
+
+<body class="flex-content-container">
+	<div class="flex-content">
+
+		<div class="content-container" id="content">
+			<div style="display: flex; flex-direction: column; row-gap: 1em;">
+
+				<h1>Edge Detection</h1>
+
+				<p>
+					For many of us, it is easy to find both the objects in a picture and their boundaries. Humans are
+					very good at identifying what something is and where it is given a clear line of sight, but what if
+					we need to find the boundaries of many objects in an enormously large number photos? While we could
+					put a room full of toddlers with markers at work and tell them to outline the edges in a stack of
```

Comment:
The reference to toddlers and morals is unnecessary. Lets rephrase this without them.

Reply:
I'd have this paragraph as:

`
For many of us, it is easy to find both the objects in a picture and their boundaries. Humans are very good at identifying what something is and where it is given a clear line of sight, but what if we need to find the boundaries of many objects in an enormously large number of photos? In computer vision, this is a general problem known as edge detection.
`